src/components/CurationPage/CurationPage.css

```diff
@@ -8,6 +8,12 @@
 
 .CurationPage .filters .text-filter-row {
   width: 100%;
+  box-sizing: border-box;
+  height: 44px;
+  background: #242129;
```

Comment:
We have a CSS variable that you can use for this color!

![image](https://user-images.githubusercontent.com/8763687/200603261-2ca5f72b-748a-410a-9ba0-827d3da3a6c4.png)